tests/neo4j/datatypes/test_temporal_types.py

```diff
@@ -204,24 +224,37 @@ def work(tx):
             return map(lambda x: getattr(x, "value", x), rec.values)
 
         self._create_driver_and_session()
+        server_supports_utc = get_server_info().has_utc_patch
```

Comment:
why is it not a simple boolean?

Reply:
ok, I found out why below.
maybe it could be renamed `server_version_may_support_utc` :P

Reply:
Yeah, for 4.4 and 4.3 there's no straight-forward way for TestKit to know if the server supports the UTC patch or not. So this value can be `1` yes, `0` no, and `0.5` maybe :grimacing:

Reply:
trilean.

Reply:
might we consider an enum?

Reply:
https://github.com/neo4j-drivers/testkit/pull/515